source/common/http/http2/protocol_constraints.cc

```diff
@@ -62,14 +62,13 @@ Status ProtocolConstraints::checkOutboundFrameLimits() {
   return okStatus();
 }
 
-Status ProtocolConstraints::trackInboundFrames(size_t length, uint8_t type, uint8_t flags,
-                                               uint32_t padding_length) {
+Status ProtocolConstraints::trackInboundFrame(uint8_t type, bool end_stream, bool is_empty) {
   switch (type) {
   case NGHTTP2_HEADERS:
   case NGHTTP2_CONTINUATION:
```

Comment:
double-checking: NGHTTP2_CONTINUATION is not expected to happen here also IIUC, yeah? (If it were, would ask to include in codec_impl.cc's end_stream condition, but because it isn't IIUC, we're good.)

Reply:
Yes, that should be true. From a frame-tracking perspective, [HEADERS + CONTINUATION sequence] are all considered one frame.